quarkchain/evm/tests/test_specials.py

```diff
@@ -20,3 +22,20 @@ def test_current_mnt_id(self):
         self.assertListEqual([result, gas_remained], [1, 4 - 3])
         self.assertEqual(len(data), 32)
         self.assertEqual(int.from_bytes(data, byteorder="big"), 2 ** 64 - 1)
+
+    def test_proc_mint_mnt(self):
```

Comment:
another test case would be some random user trying to mint native tokens but failed